Resources/Server Info/RP_Rules.txt

```diff
@@ -1,103 +1,119 @@
+
+[color=#a4885c]14.[/color] Follow escalation rules and don't murder someone for slipping you, use common sense.
+    - You can defend yourself to the extent of protecting your own life.
+    - Security is unique in escalation and can escalate to less lethal to arrest criminals as needed.
+    - Don't outright leave people to die if you get in a fight, make an effort to heal them or bring them to Medbay. A
+    - Adminhelp the situation if you think someone is over-escalating.
+    - Department strikes, revolutions (ex: cargonia and any variation thereof), or cults are antagonist-only activites and are forbidden.
+
+[color=#a4885c]15.[/color] Don't immediately ghost or suicide from your role if you do not get antagonist (referred to as "Antag-rolling").
+    - This is not fair to other players actually waiting patiently for an antagonist round. Alternatively, if you do not want to play an antagonist or do not want to cause conflict (ex: friendly antagonists), do not opt-in for antagonist roles.
+
+[color=#a4885c]16.[/color] Don't rush for or prepare equipment unrelated to your job for no purpose other then to have it "just in case" (referred to as "Powergaming").
+    - A medical doctor does not need insulated gloves, and the Head of Personnel does not need to give themselves armory access so they can go grab a gun.
+    - Don't manufacture weapons, bombs, or death poisons before you know of any reason you would need them.
+
+[color=#a4885c]17.[/color] Intentionally making yourself a major problem/annoyance/disruption for the crew or other players at large while not an antagonist is forbidden (referred to as "self-antagging").
+
+[color=#ff0000]SECURITY & COMMAND RULES[/color]
+If you regularly play Security or Command roles and got this far, I applaud you for reading.
+
+[color=#a4885c]18.[/color] If you sign up for a Command or Security role, you are expected to know the basics of the game, your job, and the job(s) you supervise, if any.
+    - Failure to know your job or how to play the game in general as Command or Security is liable to result in a job ban.
+    - As a Command or Security role, don't make friends with antagonists and trade contraband with them.
+    - Don't engage in common troublemaker behavior and lawbreaking as a Command or Security role.
+    - Do not immediately abandon your position as a Command role and go do whatever you want instead of managing your department/the station.
+
+[color=#a4885c]19.[/color] Security/Command should try to remain non-lethal and effect arrests, except in the following special circumstances, where they may choose to use lethal force:
+    - Lethal force is used against you (ex: firearms, lasers, disabling weapons with intent to kill, deadly melee weapons)
+    - Suspect is wearing clothing or showing equipment only used by enemy agents/antagonists (ex: Syndicate EVA Suit, Bloodred Hardsuit, Holoparasprite, C-20R, etc.)
```

Comment:
Maybe this is a bit too broad? Can security just lethal someone wearing a syndicate bag? Perhaps this could be restricted to syndicate weaponry (and tooling if we want to include the eva suit example, but are lethals really needed in that case?)